recipes/vit-pytorch/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "vit-pytorch" %}
+{% set version = "0.25.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2c9c71ddafbb390f0385ecd3ed728f3bc78678b5e89420c25f6360f6d4034b52
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytest-runner
+  run:
+    - python >=3.6
+    - einops >=0.3
+    - pytorch >=1.6
+    - torchvision
+
+test:
+  imports:
+    - vit_pytorch
+  source_files:
+    - tests/test.py
```

Comment:
This file isn't included in the source distribution that's uploaded to PyPI. So you either need a new release that includes it, or remove this and the commands to run pytest--and rely on the import test to check the package viability.

Reply:
yup just updated :+1: